node/src/options.rs

```diff
@@ -60,6 +60,10 @@ pub struct RunOptions {
     #[clap(long)]
     pub node_type: Option<NodeTypeConfigFile>,
 
+    /// Mock time.
+    #[clap(long)]
+    pub mock_time: Option<u64>,
```

Comment:
I don't get it. Why do we need this? Can't we set the time with rpc?

Reply:
It can be used to start the node in a "paused" state with a fixed mock time. I think it's nice to have when you need full control over the mock time. Without it, there is a potential race when some parts of the code see real system time and others see mocked time (depending on when the RPC call is made).
Bitcoin Core has "-mocktime" command line option for this:
https://github.com/bitcoin/bitcoin/blob/master/src/init.cpp#L987

Reply:
I see.

I kind of imagined the way to do this would be by setting a "ready-set-go" kind of program, where an RPC program would fire up the program after a set of preparations.

I'm OK with this for now... though I wish there's a cleaner way to do this.